gio/src/settings.rs

```diff
@@ -23,6 +35,68 @@ impl<O: IsA<Settings>> SettingsExtManual for O {
     fn set<U: ToVariant>(&self, key: &str, value: &U) -> Result<(), BoolError> {
         self.set_value(key, &value.to_variant())
     }
+
+    fn bind_with_mapping<P: IsA<glib::Object>>(
+        &self,
+        key: &str,
+        object: &P,
+        property: &str,
+        flags: SettingsBindFlags,
+        get_mapping: Option<Box<dyn Fn(&mut glib::Value, &glib::Variant) -> bool>>,
+        set_mapping: Option<Box<dyn Fn(&glib::Value, &glib::VariantType) -> glib::Variant>>,
```

Comment:
```suggestion
        set_mapping: Option<Box<dyn Fn(&glib::Value, &glib::VariantType) -> Option<glib::Variant>>>,
```

This is nullable according to the docs